contracts/interfaces/ICreate3Deployer.sol

```diff
@@ -2,15 +2,17 @@
 
 pragma solidity ^0.8.0;
 
+import { ICreate3 } from './ICreate3.sol';
+
 /**
  * @title ICreate3Deployer Interface
  * @notice This interface represents the contract responsible for deploying and initializing new contracts
  * using the `CREATE3` technique.
  */
-interface ICreate3Deployer {
-    error FailedInit();
+interface ICreate3Deployer is ICreate3 {
```

Comment:
Should we just call this `IContractDeployer` since it's basically the same for create2/create3?